styles/youtube/catppuccin.user.css

```
@@ -424,925 +441,353 @@
       --yt-button-payment-background-color: @accent-color !important;
     }
 
-    yt-icon.ytd-pinned-comment-badge-renderer * {
-      fill: @text !important;
-    }
-
-    .yt-spec-button-shape-next--size-m.yt-spec-button-shape-next--segmented-start::after {
-      background: @overlay0 !important;
-    }
-
-    .html5-ypc-purchase {
-      background: @accent-color !important;
-      color: @mantle !important;
-      &:hover {
-        background: lighten(@accent-color, 10%) !important;
-      }
-    }
-
-    .html5-ypc-description,
-    .html5-ypc-title,
-    .html5-ypc-action-heading {
-      color: @text;
+    ytd-author-comment-badge-renderer {
```

This should fix some other small issues caused by this.
```suggestion
    ytd-author-comment-badge-renderer:not([style*="--ytd-author-comment-badge-icon-background-color: transparent;]) {
```